src/PrestaShopBundle/Form/Admin/Type/CustomMoneyType.php

```diff
@@ -26,14 +26,43 @@
 
 namespace PrestaShopBundle\Form\Admin\Type;
 
+use PrestaShop\Decimal\DecimalNumber;
+use PrestaShop\PrestaShop\Core\Localization\Locale;
 use Symfony\Component\Form\AbstractTypeExtension;
 use Symfony\Component\Form\Extension\Core\Type\MoneyType;
+use Symfony\Component\Form\FormInterface;
+use Symfony\Component\Form\FormView;
 use Symfony\Component\OptionsResolver\OptionsResolver;
 
 class CustomMoneyType extends AbstractTypeExtension
 {
     public const PRESTASHOP_DECIMALS = 6;
 
+    protected const PATTERN_MAP = [
```

Comment:
for this I would like to somehow reuse PatternTransformer class :thinking:
Maybe it would require to expose some additional constants which has the patterns defined, then i can assign symfony money patterns to them :thinking: